resources/api/kio-api.yaml

```diff
@@ -139,6 +143,35 @@ paths:
                 support_url:
                   type: string
                   description: URL Where to contact for support
+                description:
+                  type: string
+                  description: Purpose of this application
+                  example: Kio manages all application base information.
+                required_approvers:
+                  type: integer
+                  description: Minimum number of approvers needed for a version. Used by fullstop. Deprecated, will be removed in the future.
+                  example: 2
```

Comment:
Not sure that we should expose this after it is deprecated. Fullstop is actually already retired since years.

Reply:
Currenlty we store this information also in Sunrise as part of [IEntitySpec](https://github.bus.zalan.do/pitchfork/sunrise/blob/79981120bf1b6d1fa07af88652f9580f66185d1e/plugins/plugin-core/typings/Entity.d.ts#L24) and I remember that in past this info was also displayed somewhere but I couldn't find it yet.